### PR TITLE
Add alternative way to login

### PR DESCRIPTION
**Pull request by kyparus**
Also fixed code indentation
Tam vse ok

### CODE REVIEW

File: vkapp/views.py [hunk 1]
```diff
@@ -30,8 +30,14 @@ def home(request):
 
 
 def login(request):
+    print ('here')
```
Comment (toniukan) [inline vkapp/views.py]: rm

File: vkapp/views.py [hunk 2]
```diff
@@ -30,8 +30,14 @@ def home(request):
...
     if request.method == 'POST':
-        access_token, user_id, expires_in = vkapi.get_auth_params_by_url(request.POST['token_url'])
+        if 'token_url' in request.POST:
+            access_token, user_id, expires_in = vkapi.get_auth_params_by_url(request.POST['token_url'])
+        else:
+            access_token, user_id, expires_in = vkapi.get_auth_params_by_login_and_password(
+                request.POST['vk_login'], request.POST['vk_pass'])
+        print ('here2')
```
Comment (toniukan) [inline vkapp/views.py]: same

File: vkapp/views.py [hunk 3]
```diff
@@ -44,6 +50,7 @@ def login(request):
             expires = datetime.now() + timedelta(seconds=int(expires_in))
         request.session['token_expire'] = expires.timestamp()
 
+        print ('here3')
```
Comment (toniukan) [inline vkapp/views.py]: :)

File: vkapp/templates/vkapp/login.html [hunk 4]
```diff
@@ -61,7 +62,39 @@ <h1 class="animated fadeInDown delay-07s">Welcome To VSY Studios</h1>
             <br>
             <br>
         </div>
-     </div>
+    </div>
+</div>
+
+<div class="main" id="welcome2">
```
Comment (toniukan) [inline vkapp/templates/vkapp/login.html]: Is it going to be a third part of the page? Will I need to scroll down to it? Please attach a screenshot.

Comment (toniukan) [reply vkapp/templates/vkapp/login.html]: Ok, I see. Just remove prints.

File: vkapp/views.py (hunk 3 above)
Comment (kyparus) [reply vkapp/views.py]: lol

File: vkapp/views.py (hunk 2 above)
Comment (kyparus) [reply vkapp/views.py]: Done!

**Pull request merged**